portfolio.html

```diff
@@ -54,5 +54,127 @@ <h2>I am a Software Developer</h2>
         </div>
       </div>
     </section>
+
+    <section class="works">
+      <div id="set1">
+        <h1>My Recent Works</h1>
+      </div>
+      <div>
+        <hr class="solid" />
+      </div>
+      <div>
+        <img
+          class="set-image"
+          src="/images/ImgPlaceholder.png"
+          alt="works image"
+        />
+      </div>
+      <div id="set2">
+        <h2>Multi-Post Stories</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+      </div>
+      <div id="lang">
+        <ul>
+          <li><img src="/images/Icons/languages/css.png" alt="csslang" /></li>
+          <li><img src="/images/Icons/languages/html.png" alt="htmllang" /></li>
+          <li>
+            <img src="/images/Icons/languages/bootstrap.png" alt="bootlang" />
+          </li>
+          <li><img src="/images/Icons/languages/ruby.png" alt="rubylang" /></li>
+        </ul>
+      </div>
+      <div class="project-button">
+        <a href="projectlink"
+          ><img
+            src="/images/Icons/Button/Enable.png"
+            alt="button
+        icon"
+        /></a>
+      </div>
+    </section>
+    <section class="project-works">
+      <div class="smallworks">
+        <h1>Professional Art Printing Data</h1>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <div>
+          <ul>
+            <li>
+              <img
+                src="/images/Icons/languages/darkicons/html.png"
+                alt="htmllang"
+              />
+            </li>
+            <li>
+              <img
+                src="/images/Icons/languages/darkicons/bootstrap.png"
+                alt="bootlang"
+              />
+            </li>
+            <li>
+              <img
+                src="/images/Icons/languages/darkicons/ruby.png"
+                alt="rubylang"
+              />
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div class="longbutton">
+        <a href="#longbotton"><img src="/images/Action.png" alt="button" /> </a>
+      </div>
+    </section>
+    <section class="about-section">
+      <div class="about">
+        <h1>About me</h1>
```

Comment:
- [ ] Well done of implementing the required changes from the other reviewer but please stick as much as you can to [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). Keep rocking 💪 

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/159082342-da126e5a-9540-4540-be2d-236bacf8964b.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/159082385-6993b368-dcac-4105-b303-209dc08df585.png)